change/@fluentui-react-dialog-4035cb74-88c2-4411-b3ac-9e1e049e2d69.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 38 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27555&commitId=76bb4c49eadc7ff0e56123f56a804cb64b380b86&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1893 screenshots unchanged, 0 screenshots with different dimensions and 38 screenshots with visible difference.


  <details><summary><b>unknown</b> 38 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Fluid Actions End.chromium.png|19398| Changed|
|Dialog.Fluid Actions Start.chromium.png|19382| Changed|
|Dialog.actions position start & position end - Dark Mode.chromium.png|15524| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|24432| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|18368| Changed|
|Dialog.actions position start & position end.chromium.png|18382| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|14849| Changed|
|Dialog.actions position start - High Contrast.chromium.png|23143| Changed|
|Dialog.actions position start - RTL.chromium.png|17514| Changed|
|Dialog.actions position start.chromium.png|17515| Changed|
|Dialog.alert - Dark Mode.chromium.png|14938| Changed|
|Dialog.alert - High Contrast.chromium.png|23506| Changed|
|Dialog.alert - RTL.chromium.png|17774| Changed|
|Dialog.alert.chromium.png|17793| Changed|
|Dialog.default - Dark Mode.chromium.png|14938| Changed|
|Dialog.default - High Contrast.chromium.png|23506| Changed|
|Dialog.default - RTL.chromium.png|17774| Changed|
|Dialog.default.chromium.png|17793| Changed|
|Dialog.nested - Dark Mode.chromium.png|11061| Changed|
|Dialog.nested - High Contrast.chromium.png|18375| Changed|
|Dialog.nested - RTL.chromium.png|13581| Changed|
|Dialog.nested.chromium.png|13581| Changed|
|Dialog.no actions - Dark Mode.chromium.png|13449| Changed|
|Dialog.no actions - High Contrast.chromium.png|18934| Changed|
|Dialog.no actions - RTL.chromium.png|14915| Changed|
|Dialog.no actions.chromium.png|14932| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|23345| Changed|
|Dialog.non-modal - High Contrast.chromium.png|30941| Changed|
|Dialog.non-modal - RTL.chromium.png|10469| Changed|
|Dialog.non-modal.chromium.png|10494| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|85395| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|102224| Changed|
|Dialog.scroll long content - RTL.chromium.png|88053| Changed|
|Dialog.scroll long content.chromium.png|88000| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|13855| Changed|
|Dialog.title custom action - High Contrast.chromium.png|19262| Changed|
|Dialog.title custom action - RTL.chromium.png|15119| Changed|
|Dialog.title custom action.chromium.png|15250| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
Definitely @spmonahan, I'll investigate! thx for the feedback